src/server/internalServer.cpp

```diff
@@ -772,7 +774,8 @@ std::unique_ptr<Response> InternalServer::handle_content(const RequestContext& r
 
   const std::shared_ptr<Reader> reader = get_reader(bookName);
   if (reader == nullptr) {
-    return Response::build_404(*this, request, bookName);
+    details = "Make a full text search for <a href=\""+m_root+"/search?pattern="+bookName+"\">"+bookName+"</a>";
```

Comment:
This skipped my attention during a previous review. The second `bookName` must be URL-decoded and HTML-escaped.